go/internal/initutil/ipfs.go

```diff
@@ -620,8 +617,8 @@ func (m *Manager) setupIPFSConfig(cfg *ipfs_cfg.Config) ([]libp2p.Option, error)
 
 	// @NOTE(gfanton): disable quic transport so we can init a custom transport
 	// with reusport disable
-	cfg.Swarm.Transports.Network.QUIC = ipfs_cfg.False
-	p2popts = append(p2popts, libp2p.Transport(quict.NewTransport, quict.DisableReuseport()))
+	cfg.Swarm.Transports.Network.QUIC = ipfs_cfg.True
```

Comment:
Is it ok the enable QUIC? Is reusePort disabled?

Reply:
that was a mistake, thanks !